### PR TITLE
Incorrect button showing on profile of someone who has sent you a friend request

### LINKED ISSUE #3505: Incorrect button showing on profile of someone who has sent you a friend request
User `@dyor` has sent me a friend request. On their profile page, the primary button in the footer is `Request to add friend` but should be `Accept friend request`.

image

### PR DESCRIPTION
**Pull request by atulya2109**
Fixes: #3505 

- Sets button text to Accept Friend Request after checking if `remoteRequested` is set for userId.

### CODE REVIEW

File: packages/message-sdk/src/components/ProfileScreen.tsx [hunk 1]
```diff
@@ -58,9 +61,18 @@ export const ProfileScreen = ({ userId }: { userId: string }) => {
       setFriendship(json.are_friends);
       setUser(json.user);
       setRequestSent(json.request_sent);
+
+      const remoteRequested: boolean = await getRemoteRequest(json.user.id);
```
Comment (hkirat) [inline packages/message-sdk/src/components/ProfileScreen.tsx:65]: you shouldn't need to send another request here. 
The request above (`${BACKEND_API_URL}/friends?userId=${userId}`) should return `remoteRequested` if it doesn't already

**Review by hkirat — CHANGES_REQUESTED** [review]

File: packages/message-sdk/src/components/ProfileScreen.tsx (hunk 1 above)
Comment (atulya2109) [reply packages/message-sdk/src/components/ProfileScreen.tsx:65]: image

This endpoint doesn't return `remoteRequested`

Comment (hkirat) [reply packages/message-sdk/src/components/ProfileScreen.tsx:65]: Can you add `remoteRequested` to the endpoing? You should be able to find the logic/code in the `backend` module

Comment (atulya2109) [reply packages/message-sdk/src/components/ProfileScreen.tsx:65]: Sure, will do.

Comment (hkirat) [reply packages/message-sdk/src/components/ProfileScreen.tsx:65]: Awesome!

Comment (atulya2109) [reply packages/message-sdk/src/components/ProfileScreen.tsx:65]: Quick question, how do i run the `backend` module locally?

Comment (hkirat) [reply packages/message-sdk/src/components/ProfileScreen.tsx:65]: cd `backend/native/backpack-api` 
yarn start

Then change the URL here to http://localhost:8080 - https://github.com/coral-xyz/backpack/blob/master/packages/common/src/constants.ts#L583

Comment (atulya2109) [reply packages/message-sdk/src/components/ProfileScreen.tsx:65]: Got it. Sorry to bother you again, but I'm getting this error when running `backpack-api` locally.

image

Comment (hkirat) [reply packages/message-sdk/src/components/ProfileScreen.tsx:65]: Can you do the following as well - 
```
cd backend/reef
docker-compose up
```

to run the database locally

Comment (atulya2109) [reply packages/message-sdk/src/components/ProfileScreen.tsx]: Ok, thanks.

**Comment by atulya2109** [comment]
Hi @hkirat, just a gentle reminder to review the requested changes.

**Pull request closed without merging**